### PR TITLE
Test

### PR DESCRIPTION
**Pull request by cornelia-schulz**

### CODE REVIEW

File: assets/stylesheets/main.css [hunk 1]
```diff
@@ -1386,7 +1381,9 @@ div#portfolio_thumbs ul li a:hover{
 	padding: 0 40px;
 	border-radius:100%;
```
Comment (AriesNg) [inline assets/stylesheets/main.css]: It makes more sense to set border-radius : 50%, except there is any other purpose?

File: pages/index.vue [hunk 2]
```diff
@@ -1,27 +1,66 @@
...
+        })
+      }
+
+       // Detects if device is on iOS 
+        const isIos = () => {
+        // eslint-disable-next-line
+        const userAgent = window.navigator.userAgent.toLowerCase()
+           return /iphone|ipad|ipod/.test( userAgent )
+        }
+        console.log(isIos())
```
Comment (AriesNg) [inline pages/index.vue]: Try to have your own console log wrapper so we can easily turn it on / off in different environment

File: static/javascripts/custom/main.js [hunk 3]
```diff
@@ -69,50 +69,23 @@ $(function ($) {
     // Text Ticker
```
Comment (AriesNg) [inline static/javascripts/custom/main.js]: Can we use Vuejs and not include any jquery?

**Comment by AriesNg** [comment]
@cornelia-schulz , can you please check how big are all bundle files' size?

File: static/javascripts/custom/main.js (hunk 3 above)
Comment (cornelia-schulz) [reply static/javascripts/custom/main.js]: All JQuery files come from a template that I have been given to work with. I can try replace content bit by bit to phase out JQuery over time.

File: assets/stylesheets/main.css (hunk 1 above)
Comment (cornelia-schulz) [reply assets/stylesheets/main.css]: I updated this to be 50%.

File: pages/index.vue (hunk 2 above)
Comment (cornelia-schulz) [reply pages/index.vue]: Ok, noted. I will try to use a wrapper in future. I have removed this console.log statement.

**Pull request merged**